yarpl/include/yarpl/flowable/Subscribers.h

```diff
@@ -120,11 +134,14 @@ class Subscribers {
           complete_(std::move(complete)) {}
 
     void onComplete() {
-      Subscriber<T>::onComplete();
-      complete_();
+      if (!userError_) { // already errored?
+        Subscriber<T>::onComplete();
```

Comment:
You can't call `onComplete()` if you've already called `onError()`.

Reply:
I think the `!userError` check does that. The unit test verifies that.